tests/scripts/expect/cli-ipmaddr.exp

```diff
@@ -40,6 +40,15 @@ expect "ff0e:0:0:0:0:0:0:1"
 expect_line "Done"
 set addr [get_ipaddr mleid]
 
+set prefix [get_meshlocal_prefix]
```

Comment:
Could you help add OTCI interface too in `openthread/tools/otci` ?@Irving-cl

Reply:
Added